src/main/java/org/rulez/demokracia/pdengine/VoteRegistry.java

```diff
@@ -64,8 +64,17 @@ public String deleteChoice(String voteId, String choiceId, String adminKey) thro
 		Vote vote = getVote(voteId);
 		vote.checkAdminKey(adminKey);
 		Choice votesChoice = vote.getChoice(choiceId);
-		vote.choices.remove(votesChoice.id);
 		
+		if(adminKey.equals("user"))
+			if(votesChoice.userName.equals(getWsUserName()))
+				if(vote.canAddin)  
+					vote.choices.remove(votesChoice.id);
```

Comment:
uncovered code

Reply:
Strange. I missed something on the tests.

Reply:
Fixed